agents/ls-java/src/main/resources/installers/1.0.1/org.eclipse.che.ls.java.script.sh

```diff
@@ -26,7 +26,8 @@ unset PACKAGES
 command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
 command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
 
-AGENT_BINARIES_URI=http://download.eclipse.org/jdtls/snapshots/jdt-language-server-latest.tar.gz
+DOWNLOAD_AGENT_BINARIES_URI='${WORKSPACE_MASTER_URI}/agent-binaries/jdt.ls.tar.gz'
```

Comment:
it's the black magic of single quotes.

Reply:
WORKSPACE_MASTER_URI can be changed before using DOWNLOAD_AGENT_BINARIES_URI. 
I think is not license issue here because I take already prepared bundle. @skabashnyuk @vparfonov Am I right?

Reply:
@tsmaeder both projects are Eclipse project, what licensing issue do you mean? Who can help us with this questions?

Reply:
I talked to @gorkem and I guess anything that is not from the Eclipse foundation itself has been approved for use in Eclpse projects. We should be fine!